docs/pages/api-docs/badge-unstyled.json

```diff
@@ -63,7 +63,7 @@
   "forwardsRefTo": "HTMLSpanElement",
   "filename": "/packages/material-ui-unstyled/src/BadgeUnstyled/BadgeUnstyled.js",
   "inheritance": null,
-  "demos": "<ul><li><a href=\"/components/badges/\">Badges</a></li></ul>",
+  "demos": "<ul><li><a href=\"/components/avatars/\">Avatars</a></li>\n<li><a href=\"/components/badges/\">Badges</a></li></ul>",
```

Comment:
TODO: Is the addition of Avatar here intended?

Reply:
@mnajdova I guess the idea was to only list the demo of the styled component? My question would be why we special case the unstyled component in the first place i.e. just list `BadgeUnstyled` in `/components/badges`?

Reply:
BadgeUnstyled is only referenced on the badge demo page. I fail to see the link with the avatar, outside of this demo: https://next.material-ui.com/components/avatars/#with-badge. It looks like a regression.

Reply:
But at the same time it is already listed in the JSDOC for the Badge:
https://github.com/mui-org/material-ui/blob/b8bb45119ce0a2f97b70e361a4ac7179f6f8ad1a/packages/material-ui-unstyled/src/BadgeUnstyled/BadgeUnstyled.d.ts#L127-L128

Opening a separate PR to explicitly list these demos instead of trying to be too clever about it.

Reply:
Resolved by https://github.com/mui-org/material-ui/pull/25900